jersey-tck/pom.xml

```diff
@@ -67,10 +145,140 @@
                         </goals>
                         <configuration>
                             <dependenciesToScan>jakarta.ws.rs:jakarta.ws.rs-tck</dependenciesToScan>
+                            <includes>
+                                 <include>**/tck/**/*IT</include>                            
```

Comment:
We should rather stick with the default and exclude what we don't want to have scanned.

Reply:
I have renamed the test files to *IT.java, we might not need to specify this default inclusion.